sdk/schemaregistry/Azure.Data.SchemaRegistry/src/SchemaFormat.cs

```diff
@@ -11,18 +11,38 @@ namespace Azure.Data.SchemaRegistry
     {
         private readonly string _value;
 
+        private const string AvroValue = "Avro";
+        private const string JsonValue = "JSON";
+        private const string CustomValue = "Custom";
+
         /// <summary> Initializes a new instance of <see cref="SchemaFormat"/>. </summary>
         /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
         public SchemaFormat(string value)
         {
             _value = value ?? throw new ArgumentNullException(nameof(value));
-            ContentType = $"application/json; serialization={_value}";
+            switch (value)
```

Comment:
nit: I'd suggest leaving the ContentType property off of here and just add an internal method, ToContentType that returns the correct ContentType.